### DISCUSSION TITLE
Keep Payee and Category same when launching with Empty Combo Box enter launch

### DISCUSSION THREAD

**Discussion by SJK55** [moneymanagerex/moneymanagerex#5015] [Ideas]
Suggestion - Make that clear, not a bug.
A Suggestion.

However when starting a new transaction upon hitting empty combo box on Payee the cursor lands in the 'smart search box'
Great!
Now when you do the same in the Category, it doesn't follow the same pattern. ie The new box that opens lands on the 'tree'
It would be great if the Categories went to the Search box like the Payees does!
The same inconsistency (as you would expect) appear is the New Recurring Transaction Combo box. (I suppose similar call to a library etc)
![Payees Smart Search Deafult](https://user-images.githubusercontent.com/14268231/187872918-2118bacc-1f9e-4bcb-83a3-cbdbc295a2d7.jpg)
![Category Search Deafult](https://user-images.githubusercontent.com/14268231/187872932-7f0b29fe-8f0e-452d-bed8-8e56ca52bd08.jpg)

**Comment by tactilis** [comment]
Current behaviour as described by @SJK55 is confirmed.

This is a good idea. Enhancement Request #5016 raised to track it

**Reply by SJK55** [reply]
Thank you.

**Comment by tactilis** [comment]
@SJK55

@whalley has fixed a bug that was preventing focus being on the Search field when the dialog is opened.

This fix will be in 1.5.20 when that is released.

**Reply by SJK55** [reply]
Excellent!
